buildenv/jenkins/jobs/infrastructure/omrMirror

```diff
@@ -77,4 +77,9 @@ pipeline {
             }
         }
     }
-}
\ No newline at end of file
+    post {
+        failure {
+            slackSend channel: '#jenkins', color: 'danger', message: "Failed: ${env.JOB_NAME} #${env.BUILD_NUMBER} (<${env.BUILD_URL}|Open>)"
```

Comment:
Does this pick up the same pipeline configuration or does it need to be told which Slack instance in some other way?

Reply:
and that the slack is picked up from the Jenkins config.